app/controllers/users_controller.rb

```diff
@@ -30,6 +30,21 @@ def add_contact
     redirect_to home_index_path
   end
 
+  def search
+     if params[:q].blank?
+        @results = User.all
+     else
+        search_pattern = params[:q].downcase
+        @results = []
+        User.all.each do |user|
+
```

Comment:
is this newline on purpose? It somehow makes it harder to see the `do` block

Reply:
This code doesn't exist anymore